pkgs/tools/audio/playerctl/default.nix

```diff
@@ -14,6 +14,10 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ meson ninja pkgconfig gtk-doc docbook_xsl gobject-introspection ];
   buildInputs = [ glib ];
 
+  mesonFlags = [ "-Dbash-completions=true" ];
+
+  PKG_CONFIG_BASH_COMPLETION_COMPLETIONSDIR= "${placeholder "out"}/share/bash-completion/completions";
```

Comment:
is this line necessary? I get the same output without it

Reply:
I copied the line from https://github.com/NixOS/nixpkgs/blob/6be09e27c935567b4a6f77c6772acbc971e860fd/pkgs/tools/misc/colord/default.nix#L95-L99

I observed the same behaviour but forgot to mention it in the PR.

Reply:
I think we can remove this line